docs/developer-docs/smart-contracts/advanced-features/time-and-timestamps.mdx

```diff
@@ -52,13 +52,10 @@ let current_time = IC_API::time();
 <AdornedTab value={"typescript"} label="TypeScript" endAdornment={<BetaChip />}>
 
 ```typescript
-import { Canister, ic, nat64, query } from 'azle/experimental';
-
-export default Canister({
-    time: query([], nat64, () => {
-        return ic.time();
-    })
-});
+@query([], IDL.Nat64)
+time(): bigint {
+    return time();
```

Comment:
Similar to a previous comment, I think this should show the containing class and the azle imports, it's sometimes very difficult to find out where these APIs are coming from and how to actually use them.

Reply:
revised example